### PR TITLE
Changes for HyperV-Socket agent's CAs certificates

### PR DESCRIPTION
**Pull request by algamaes**

### CODE REVIEW

File: rpc/nodeagent/security/certificate/moc_nodeagent_certificate.proto [hunk 1]
```diff
@@ -34,8 +34,28 @@ message Certificate {
...
+	repeated CertificateSigningRequest CSRs = 1;
+}
+
+message CertificateSigningRequest {
+    string name = 1;
+    string csr = 2 [(sensitive) = true];
+    string oldCertificate = 3 [(sensitive) = true];
+    Status status = 4;
+    string caName = 5;
+    string groupName = 6;
```
Comment (raghavendra-nataraj) [inline rpc/nodeagent/security/certificate/moc_nodeagent_certificate.proto:47]: Can we remove the unnecessary fields in the struct

**Review by raghavendra-nataraj — APPROVED** [review]

**Comment by raghavendra-nataraj** [comment]
Can you check why the tests are failing?

**Comment by algamaes** [comment]
@raghavendra-nataraj I had missing cherry-picking a fix from another branch I had, sorry. Let's not merge this PR until node's PR is approved, because the new methods added to node certificates will fail the builds until they're implemented.

**Comment by raghavendra-nataraj** [comment]
Can we introduce stubs to avoid the build issue?

**Review by manoj-kadam — APPROVED** [review]

**Pull request closed without merging**